Data/SessionService.cs

```diff
@@ -10,19 +10,56 @@ public sealed class SessionService
     public SessionService(IDbConnection dbConnection) =>
         _dbConnection = dbConnection;
 
-    public IReadOnlyList<Session> GetSessions() =>
-        _dbConnection.Query<Session, Author, Session>(
-            @"SELECT s.id AS sessionId, s.when, a.id AS authorId, a.name, a.surname 
-                FROM session s
-                JOIN author a
-                    ON s.chair_id = a.id",
-            (s, a) =>
+    public IReadOnlyList<Session> GetSessions() {
+
+        var paperDictionary = new Dictionary<int, Paper>();
+
+        PaperService t = new PaperService(_dbConnection);
```

Comment:
Niedobrze, to powinno się robić dependency injection. Odsyłam do https://mimuw.edu.pl/~mg394302/bd2021-csharp.html#inversion-of-control-i-dependency-injection.